### DISCUSSION TITLE
Inquiry about BC in CH model

### DISCUSSION THREAD

**Discussion by abc-hy** [idaholab/moose#20410] [Q&A Modules: Phase field] [answered]
Hi everyone,

I have a question about the boundary condition. If I don't set the boundary condition in the input file, what is the boundary condition that will be applied by default?

Best regards,
Haiying

**Comment by dschwen** [comment, marked as answer]
No flux

**Reply by abc-hy** [reply]
So, this is for left, right, top and down boundary?

**Reply by abc-hy** [reply]
For the concentration on the top and bottom boundary, what value should I set?

**Reply by GiudGiud** [reply]
any boundary that does not have a boundary condition will effectively have a no-flux BC.
There's more than one CH model. Which one are you talking about?
